srcpkgs/spotify/template

```diff
@@ -1,21 +1,67 @@
 # Template file for 'spotify'
 pkgname=spotify
 version=1.1.26
-revision=1
+revision=2
+_ver="${version}.501.gbe11e53b-15_amd64"
+_filename="spotify-client_${_ver}.deb"
 archs="x86_64"
 create_wrksrc=yes
 build_style=fetch
-depends="binutils gtk+ nss GConf libXScrnSaver libatomic"
+hostmakedepends="curl w3m"
+depends="GConf"
 short_desc="Proprietary music streaming client"
 maintainer="Stefan Mühlinghaus <jazzman@alphabreed.com>"
-license="Proprietary"
+license="custom:Proprietary"
 homepage="https://www.spotify.com"
+distfiles="http://repository.spotify.com/pool/non-free/s/spotify-client/${_filename}"
+checksum="54ba1bd91020913273c469591cedeebfa34aedc250033dde7c8b3fded94a9c71"
+_license_checksum="3949b3442f9e17e6934502884caa1e78fa97c48c4b8b79c2de8eb2c301b69f3f"
 repository=nonfree
-_sversion=".501.gbe11e53b-15_amd64"
-_schecksum="54ba1bd91020913273c469591cedeebfa34aedc250033dde7c8b3fded94a9c71"
+restricted=yes
+nostrip=yes
+
+post_extract() {
+	curl -L https://www.spotify.com/us/legal/end-user-agreement/ | \
+		w3m -dump -I utf-8 -T text/html > LICENSE
+	echo "$_license_checksum  LICENSE" | sha256sum -c || { sha256sum LICENSE; return 1; }
```

Comment:
This command is fine as is.
But, I prefer to use `xbps-digest` in the same way `common/hooks/do-fetch/00-distfiles.sh` do it.

Reply:
Done, I think. Thanks!